PWGHF/D2H/Tasks/taskXicToXiPiPi.cxx

```diff
@@ -209,11 +267,13 @@ struct HfTaskXicToXiPiPi {
       registry.fill(HIST("hCPAxyLambda"), candidate.cosPaLambda(), ptCandXic);
       registry.fill(HIST("hMassXiPi1"), candidate.invMassXiPi0(), ptCandXic);
       registry.fill(HIST("hMassXiPi2"), candidate.invMassXiPi1(), ptCandXic);
+      registry.fill(HIST("hChi2geoXi"), candidate.kfCascadeChi2(), ptCandXic);
+      registry.fill(HIST("hChi2geoLam"), candidate.kfV0Chi2(), ptCandXic);
     } // candidate loop
   }   // process
 
   /// MC analysis and fill histograms
-  void processMc(soa::Filtered<soa::Join<aod::HfCandXic, aod::HfSelXicToXiPiPi, aod::HfCandXicMcRec>> const& candidates,
+  void processMc(soa::Filtered<soa::Join<aod::HfCandXic, aod::HfCandXicKF, aod::HfSelXicToXiPiPi, aod::HfCandXicMcRec>> const& candidates,
```

Comment:
@creetz16 This adds dependency on KFParticle without the possibility to analyse candidates reconstructed with the DCA fitter. Please support both versions.

Reply:
Tagging @JaeYoonCHO for info.

Reply:
Hi @vkucera Sorry for the late reply. I followed this up here: https://github.com/AliceO2Group/O2Physics/pull/7277